Source/Core/Core/IOS/VersionInfo.cpp

```diff
@@ -9,334 +9,339 @@
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         58,         0x3a1820,   0x030110,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         59,         0x3b2421,   0x101811,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         60,         0x3c181e,   0x112408,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         61,         0x3d161e,   0x030110,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         62,         0x3e191e,   0x022712,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         70,         0x461a1f,   0x060309,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         80,         0x501b20,   0x030310,         MEM1_SIZE,
-         MEM1_SIZE,  MEM1_END,   MEM1_ARENA_BEGIN, MEM1_ARENA_END,
-         MEM2_SIZE,  MEM2_SIZE,  0x93600000,       MEM2_ARENA_BEGIN,
-         0x935E0000, 0x935E0000, 0x93600000,       HOLLYWOOD_REVISION,
-         RAM_VENDOR, 0x93600000, 0x93620000,       0,
-     },
-     {
-         257,
-         0x707,
-         0x82209,
-         MEM1_SIZE,
-         MEM1_SIZE,
-         MEM1_END,
-         MEM1_ARENA_BEGIN,
-         MEM1_ARENA_END,
-         MEM2_SIZE,
-         MEM2_SIZE,
-         0x93600000,
-         MEM2_ARENA_BEGIN,
-         0x935E0000,
-         0x935E0000,
-         0x93600000,
-         HOLLYWOOD_REVISION,
-         RAM_VENDOR_MIOS,
-         PLACEHOLDER,
-         PLACEHOLDER,
-         PLACEHOLDER,
-     }}};
+std::array<MemoryValues, 41> ios_memory_values;
```

Comment:
Maybe consider updating SetupMemory (in IOS) instead? These values are supposed to be dumped directly from IOS kernels.